packages/doctype-tile/src/tile-doctype.ts

```diff
@@ -264,4 +183,42 @@ export class TileDoctype<T = Record<string, any>> extends Doctype {
         return _signDagJWS(commit, did, this.controllers[0])
     }
 
+    /**
+     * Create genesis commit.
+     * @param did - DID making (and signing) the commit
+     * @param content - genesis content
+     * @param metadata - genesis metadata
+     */
+    static async makeGenesis<T>(did: DID, content: T | null, metadata?: TileMetadataArgs): Promise<CeramicCommit> {
```

Comment:
@stbrody I am too late to the party, but maybe it returns `Promise<SignedCommitContainer>`

Reply:
I'll see if I can do that in a follow up, but I think it'd need to be `Promise<SignedCommit|SignedCommitContainer>` since the genesis commit can come back unsigned if there's no content

Reply:
Oomps, did not know that. My suggestion then does not make sense.